Policies/github-terms-of-service.md

```diff
@@ -36,95 +36,118 @@ Thank you for using GitHub! We're happy you're here. Please read this Terms of S
+- You must be a human to create an Account. Accounts registered by "bots" or other automated methods are not permitted. We do permit machine accounts. When you need to deal with, or know about, machine accounts, make sure you have read the following separately bullet-pointed text (otherwise you can skip it):  
+    * A machine account is an Account set up by an individual human who accepts the Terms on behalf of the Account, provides a valid email address, and is responsible for its actions. A machine account is used exclusively for performing automated tasks. Multiple users may direct the actions of a machine account, but the owner of the Account is ultimately responsible for the machine's actions. You may maintain no more than one free machine account in addition to your free User Account.
+- One person or legal entity may maintain no more than one free Account (you can choose to control a machine account as well, but it can only be used for running a machine).
+- You must be age 13 or older. This requirement is related to our compliance with United States law. GitHub does not target our Service to children under 13, and we do not permit any Users under 13 on our Service. If we learn of any User under the age of 13, we will [terminate that User’s Account immediately](#l-cancellation-and-termination). If you are a resident of a country outside the United States, your country’s minimum age may be older; in such a case, you are responsible for complying with your country’s laws.
 - Your login may only be used by one person — i.e., a single login may not be shared by multiple people. A paid Organization  may only provide access to as many User Accounts as your subscription allows.
-- You may not use GitHub in violation of export control or sanctions laws of the United States or any other applicable jurisdiction. You may not use GitHub if you are or are working on behalf of a [Specially Designated National (SDN)](https://www.treasury.gov/resource-center/sanctions/SDN-List/Pages/default.aspx) or a person subject to similar blocking or denied party prohibitions administered by a U.S. government agency.  GitHub may allow persons in certain sanctioned countries or territories to access certain GitHub services pursuant to U.S. government authorizations.  For more information, please see our [Export Controls policy](https://help.github.com/en/articles/github-and-export-controls).  
+- You may not use GitHub in violation of export control or sanctions laws of the United States or any other applicable jurisdiction. You can read the following bullet point for more information (if you don't need more information, you can just skip it):
```

Comment:
Is `you may not use` repeated. Can be deduplicated.

Reply:
It is repeated but I can't see how it can be de-duplicated in an effective way. The nature of the sub bullet point isn't simply a series of 'you may not' statements.

The tip given by @KOLANICH in https://github.com/github/site-policy/pull/204#discussion_r354987952 is a proper tip also for the text here.

Reply:
`You may not use GH for: `, then a list.

Reply:
Well, you've voiced disagreement with 'you can skip' texts, so perhaps because of that we disagree here.

If you remove the 'you can skip' device, your suggestion isn't so bad--you can do some de-duplication.

However, if the 'you can skip' device remains (in some fashion), I don't think your suggested de-duplication helps. The sub bullet point only has one 'You may not use' statement.

Reply:
Collapsible-text tip has now been implemented here, in the patch as a new commit. A new PR possibly needs to be made for the commit.